internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/model/ScanContext.java

```diff
@@ -68,6 +74,52 @@
     // Report template id to generate report
     private String reportTemplateId;
 
+    // Authentication builder type
```

Comment:
```suggestion
    // Authentication builder type.
```
Same with other lines.